src/app/index.scss

```diff
@@ -7,7 +7,7 @@
 
 	color-scheme: light dark;
 	color: rgba(255, 255, 255, 0.87);
-	background-color: #242424;
+	background-color: #fff;
```

Comment:
not sure about that...

Reply:
Took this one from figma as long as all of the backgrounds on pages are #ffffff.